docs/src/Form/index.js

```diff
@@ -28,6 +28,7 @@ class FormExample extends React.Component {
       {
         fieldType: 'password',
         name: 'Password',
+        helpBlock: 'If you want to make your password strong you could use a password manager',
```

Comment:
Not a fan of this wording - the tone is at odds with what we're using elsewhere.